docs/advanced_reference/commands/autonomy_mint.md

```diff
@@ -38,8 +38,14 @@ This command group consists of a number of functionalities to mint components, a
 
     The options `--use-ethereum`, `--use-goerli`, `--use-custom-chain` and `--use-local` are mutually exclusive.
 
-`--skip-hash-check`
-: Skip hash check when verifying dependencies on chain.
+`-t, --timeout FLOAT`
+: Timeout for verifying emitted events
```

Comment:
Please add more information here. What events? What does this timeout mean in a mint command?